src/applications/pensions/config/chapters/04-household-information/currentSpouseFormerMarriages.jsx

```diff
@@ -139,6 +139,7 @@ export default {
             'locationOfMarriage',
             'locationOfSeparation',
           ],
+          /* What should we do about this object? */
```

Comment:
The overall spouseMarriages object, or the reasonForSeparation? Currently on vets-json-schema, we use the default marriages schema for the spouseMarriages (https://github.com/department-of-veterans-affairs/vets-json-schema/blob/8147f9b8a49c2dc5491118f37ec8dc267036af20/src/schemas/21P-527EZ/schema.js#L348 and https://github.com/department-of-veterans-affairs/vets-json-schema/blob/8147f9b8a49c2dc5491118f37ec8dc267036af20/src/common/definitions.js#L748). There are ways to customize the default definitions in vets-json-schema if we wanted to go down that route to be more precise, since the version there is more generic than ours